### PR TITLE
 Adding modulo and inverse modulo caclulator, Resolving issue #99

### LINKED ISSUE #99: Modulo and Inverse Modulo Calculator
## Description

This calculator will find out modulus and inverse modulus.

For ex:

7 mod 3=1
5^(-1) = 9 mod 11

@vasu-1 please assign this issue to me.

### Contribution Event

GSSOC22

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/vasu-1/CalcHub/blob/main/.github/ContributingGuidelines.md) & [Code of conduct](https://github.com/vasu-1/CalcHub/blob/main/.github/CODE_OF_CONDUCT.MD) of this project.

### PR DESCRIPTION
**Pull request by AnIkeT126**
## Description 📜

This PR adds a calculator that aims to calculate the modulus and inverse modulus of two given numbers and is made using HTML, CSS & JS.

Fixes #99, under GSSoC'22

<hr>
 
## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/vasu-1/CalcHub/blob/main/.github/ContributingGuidelines.md) & [Code of conduct](https://github.com/vasu-1/CalcHub/blob/main/.github/CODE_OF_CONDUCT.MD) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GSSOC'22 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

image

image

image

image

### CODE REVIEW

**Comment by AnIkeT126** [comment]
@vasu-1 can you review this PR, this is in accordance will all the guidelines and contains the code file along with a readme that contains the screenshots of the output.

**Review by mansi-garg-12 — APPROVED** [review]
Good to go checked!

File: Calculators/Modulo_Inverse_Modulo_Calculator/index.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Affine Cipher</title>
```
Comment (vasu-1) [inline Calculators/Modulo_Inverse_Modulo_Calculator/index.html:5]: change title

File: Calculators/Modulo_Inverse_Modulo_Calculator/readme.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+![image](https://user-images.githubusercontent.com/82095877/157040325-fcb0e630-583c-4fb2-af0c-4f4936364a27.png)
+
+
+![image](https://user-images.githubusercontent.com/82095877/157039611-372ff927-6780-4386-93fc-3324135a0ff4.png)
+
+![image](https://user-images.githubusercontent.com/82095877/157040418-b1be4345-f46a-49ae-9def-26f049b9ef00.png)
+
+![image](https://user-images.githubusercontent.com/82095877/157040727-c03e8096-4c0e-4382-8f2e-7da5ad415819.png)
+
```
Comment (vasu-1) [inline Calculators/Modulo_Inverse_Modulo_Calculator/readme.md:32]: add only two screenshots !!

**Review by vasu-1 — CHANGES_REQUESTED** [review]

**Comment by AnIkeT126** [comment]
I have updated the changes in a new PR

**Pull request closed without merging**